README.md

```diff
@@ -16,7 +16,10 @@ the Sia network.
 
 ## Prerequisites
 
-- [node & npm (packaged together)](https://nodejs.org/download/)
+- [siad](https://github.com/NebulousLabs/Sia)
+- [node & npm 6.9.0 LTS](https://nodejs.org/download/)
+Earlier node versions may work, but they do not have guarenteed support.
+- `libxss` is required for electron on some systems.
 
```

Comment:
I would go as far as saying 'On debian, libxss is required as an electron dependency. This can be installed with `sudo apt-get install libxss1`'. The user shouldn't need to figure out what package they need for libxss. Also, better I think to be more specific than 'some systems', as people on uncommon distros usually understand that they could run into similar problems. Our documented use cases should cover all common distros, we will see more issues reported if we are missing things for Ubuntu, fedora, etc.